central/cve/matcher/matcher.go

```diff
@@ -154,6 +146,24 @@ func (m *CVEMatcher) IsClusterAffectedByIstioCVE(ctx context.Context, cluster *s
 	return false, nil
 }
 
+// GetValidIstioVersions is to check if Istio is running and get all Istio versions
```

Comment:
nit: maybe `// GetValidIstioVersions returns all running Istio versions in the given cluster, if there are any, and nil otherwise`